### PR TITLE
use updated_timestamp instead of created_timestamp in incremental logic

### PR DESCRIPTION
**Pull request by fivetran-jamie**
Pull Request
**Are you a current Fivetran customer?** 

fivetran made PR

**What change(s) does this PR introduce?** 

- [X] adjusts logic in orders_attribution model to transform newly updated orders in addition to newly created ones by using updated_timestamp instead of created_timestamp for is_incremental comparisons. Every order has an updated_timestamp
- [X] updates changelog
- [X] updates dbt_project.yml
- [X] adds note about incremental materialization to readme and docs
- [X] update requirements.txt
- [X] tweak PR template
- [x] regen docs 
- [X] make postgres use the `delete+insert` incremental_strategy, as it does not have `merge`.  in previous versions of dbt if you provided a unique_key and merge as the strategy in postgres, no error was raised, but that has recently changed (i believe changing the `requirements.txt` lead to me uncovering this)

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

This is breaking because people WILL need to run a full refresh.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature https://github.com/fivetran/dbt_shopify_holistic_reporting/issues/8
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
💅 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-dbt-redshift~=1.0.0
-dbt-postgres~=1.0.0
-dbt-spark~=1.0.0
-dbt-spark[PyHive]~=1.0.0
+dbt-snowflake>=1.0.0
+dbt-bigquery>=1.0.0
+dbt-redshift>=1.0.0
+dbt-postgres>=1.0.0
+dbt-spark>=1.0.0
+dbt-spark[PyHive]>=1.0.0
```
Comment (fivetran-catfritz) [inline integration_tests/requirements.txt:6]: I can't remember what/if we decided as a group, but should we also add `,<2.0.0` for each adapter?

File: packages.yml [hunk 2]
```diff
@@ -2,5 +2,8 @@ packages:
   - package: fivetran/shopify
     version: [">=0.6.0", "<0.7.0"]
 
-  - package: fivetran/klaviyo
-    version: [">=0.4.0", "<0.5.0"]
+  # - package: fivetran/klaviyo
+  #   version: [">=0.4.0", "<0.5.0"]
+
+  - git: https://github.com/fivetran/dbt_klaviyo.git
+    revision: postgres-incremental-strategy
```
Comment (fivetran-catfritz) [inline packages.yml:9]: Standard note to update before merge.

File: integration_tests/requirements.txt (hunk 1 above)
Comment (fivetran-jamie) [reply integration_tests/requirements.txt:6]: ah yeah

**Review by fivetran-catfritz — APPROVED** [review]
LGTM!

**Pull request merged**